### ISSUE TITLE
NVDA announces indeterminate progressbar as half checked

### ISSUE THREAD

**Issue by adamsamec** [nvaccess/nvda#13910]
### Steps to reproduce:
1. Visit [this demo page][example].
2. Make sure that browse mode of NvDA is enabled.
3. Press the "1" key to jump to the 'Accessibility tests of the "progressbar" ARIA role' heading.
4. Read through the page using the Down arrow key until you reach the "End of examples" button.

### Actual behavior:
Below each of the &lt;h2&gt; heading, is an element with the "progressbar" role. The first example progressbar element, which is a determinate progressbar, is announced as expected. However, the second example progressbar, which is an indeterminate progressbar, is announced as "Updating... progress bar half checked   0".

### Expected behavior:
The second example progressbar should be announced by NVDA as "Updating... determinate progress bar".

#### NVDA installed/portable/running from source:
Installed.

#### NVDA version:
NVDA 2022.1.

#### Windows version:
Windows 11, version 21H2 (OS Build 22000.795)

#### Name and version of other software in use when reproducing the issue:
Google Chrome, version 103.0.5060.114

#### Does the issue still occur after restarting your computer?
Yes.

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No.

#### If NVDA add-ons are disabled, is your problem still occurring?
Yes.

#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes.

[example]: https://jsbin.com/vawemabeji/edit?html,output

**Comment by CyrilleB79** [comment]
Hi @adamsamec

This issue is the same described in #10644.
This has been fixed in NVDA 2022.2, presently at RC stage.
Would you mind test NVDA 2022.2rc1 and check that the fix suits your needs?
Thanks.

**Comment by adamsamec** [comment]
[@CyrilleB79](https://github.com/CyrilleB79), sorry for the duplicate. When I was searching whether the issue already exists, I used the keyword "progressbar" instead of "progress bar", so couldn't find it.

I have tested NVDA 2022.2rc1, and it looks like it workds well. However, for a better consistency with a determinate progressbar, I would name the indeterminate progressbar not as "busy indicator" but rather "busy progressbar". But that's just my subjective viewpoint.

**Comment by CyrilleB79** [comment]
No problem with the duplicate, it happens frequently.

If you want to suggest renaming this control, please read what has already been discussed in #10644 and #13428. And then rewrite the initial description of this issue accordingly (or open a new one).

If you feel renaming this control is not so important, just close this issue.

Thanks.
